app/controllers/sessions_controller.rb

```diff
@@ -92,6 +98,9 @@ def set_auth_session_vars(omniauth)
     session[:provider] = omniauth.provider
     session[:uid] = omniauth.uid
     session[:username] = omniauth.try(:username) || omniauth.try(:email)
+    if omniauth.is_a?(GoogleAuth)
+      session[:email_for_google] = omniauth.try(:email)
+    end
```

Comment:
Same as above, you can one-line this as:
```ruby
session[:email_for_google] = omniauth.try(:email) if omniauth.is_a?(GoogleAuth)
```

Reply:
This one fits in the line length limit -- done.